packages/login/test/login-overlay.common.js

```diff
@@ -221,14 +221,14 @@ describe('title slot', () => {
   });
 });
 
-describe('custom-fields slot', () => {
+describe('custom-form-area slot', () => {
```

Comment:
Do you think it makes sense to also update the test names here?

Reply:
Renamed.